arma3/@task_force_radio/addons/task_force_radio/functions/common.sqf

```diff
@@ -21,7 +21,7 @@ TF_CODE_OFFSET = TF_SW_STEREO_OFFSET + 1;
 TF_ADDITIONAL_CHANNEL_OFFSET = TF_CODE_OFFSET + 1;
 TF_ADDITIONAL_STEREO_OFFSET = TF_ADDITIONAL_CHANNEL_OFFSET + 1;
 
-TF_LR_SPEAKER_OFFSET = 7;
+TF_LR_SPEAKER_OFFSET = 8;
```

Comment:
According to script.h its not. but the radio owner is at 7 now.. apparently..
Which makes sense because it actually was there before in generateSwSettings line 35
But I dont understand why you are setting that to "nil" in generateLrSettings. Why not just use the same getPlayerUID as in SwSettings